packages/medusa/src/utils/feature-flag-decorators.ts

```diff
@@ -0,0 +1,69 @@
+import { getConfigFile } from "medusa-core-utils"
+import { Column, ColumnOptions, Entity, EntityOptions } from "typeorm"
+import featureFlagsLoader from "../loaders/feature-flags"
+import path from "path"
+import { ConfigModule } from "../types/global"
+
+export function FeatureFlagColumn(
+  featureFlag: string,
+  columnOptions: ColumnOptions
+): PropertyDecorator {
+  const { configModule } = getConfigFile(
```

Comment:
suggestion: Since this is a common part of the next decorator, can we move that in a separate non exported function such as
```ts
function getFeatureFlagRouter(): FlagRouter {
  const { configModule } = getConfigFile(
    path.resolve("."),
    `medusa-config`
  ) as { configModule: ConfigModule }

 return featureFlagsLoader(configModule)
}
```